pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -391,27 +403,32 @@ class PauliY(Observable, Operation):
 
     _queue_category = "_ops"
 
-    def __init__(self, wires=None, id=None):
+    def __init__(self, wires: Optional[WiresLike] = None, id: Optional[str] = None):
```

Comment:
```suggestion
    def __init__(self, wires: WiresLike, id: Optional[str] = None):
```
Optional while we're here?